web-client/src/ducks/myRequests/functions.ts

```diff
@@ -0,0 +1,29 @@
+import { Post } from 'src/models/posts';
```

Comment:
Split ducks/posts to ducks/myRequests and ducks/myOffers.  Followed ducks patterns established in posts.